From 5df1555828add4e256cb4bba49fffffc22127c34 Mon Sep 17 00:00:00 2001
From: Dave Watson <dade.watson@gmail.com>
Date: Wed, 27 May 2020 08:21:04 -0700
Subject: [PATCH] aarch64: Fix __sigset build issue on muslC

Described in issue #176.   Taken from Alpine Linux project.

Change __sigset_t to sigset_t

Upstream-Status: Backport [https://github.com/libunwind/libunwind/commit/5df1555828]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 include/libunwind-aarch64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/libunwind-aarch64.h b/include/libunwind-aarch64.h
index db01a9f69..d1bd8708f 100644
--- a/include/libunwind-aarch64.h
+++ b/include/libunwind-aarch64.h
@@ -202,7 +202,7 @@ typedef struct
 	unsigned long uc_flags;
 	struct ucontext *uc_link;
 	stack_t uc_stack;
-	__sigset_t uc_sigmask;
+	sigset_t uc_sigmask;
 	struct unw_sigcontext uc_mcontext;
   } unw_tdep_context_t;
 
